cli/tests/pcluster/validators/test_fsx_validators.py

```diff
@@ -438,3 +439,15 @@ def test_auto_import_policy_validator(mocker, auto_import_policy, cluster_region
 
     actual_failures = FsxAutoImportValidator().execute(auto_import_policy, "s3://test/test1/test2")
     assert_failure_messages(actual_failures, expected_message)
+
+
+def test_fsx_non_happy_ontap_openfsx_mounting(mocker):
```

Comment:
Let's remember to remove this unit tests covering the newly added validator, as this validator should not be added to production.

Reply:
Yes